test/gen_yaml_and_log.rb

```diff
@@ -0,0 +1,129 @@
+require "erb"
```

Comment:
I'm lost. What is the point of this file? Sorry!

Reply:
It generates the babeltrace log and the yaml matching the log. Then we can use the script you did to generate the source, and compare the output with the babeltrace log (generated). Well I just came out with another question, if the order on which we push messages is guaranteed I expect the output to match if not, we are done with this approach and need to check another.

Reply:
Oh I see, but we don;t need to generate the `yaml matching log` I think. What is the point of this file?